src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Block {
+
+    private final String BLOCK;
```

Comment:
Why is this variable final? If the block number change and without a setter method can we change it now? Check it out